source/docs/software/hardware-apis/sensors/gyros-software.rst

```diff
@@ -236,6 +280,38 @@ The following example shows how to stabilize heading using a simple P loop close
             drive.TankDrive(.5 + kP * error, .5 - kP * error);
         }
 
+    .. code-block:: python
+
+        from wpilib import Spark
+        from wpilib import MotorControllerGroup
+        from wpilib.drive import DifferentialDrive
+
+        # Use gyro declaration from above here
+
+        # The gain for a simple P loop
+        kP = 1
```

Comment:
All of this needs to be in `robotInit`, and they all need to be assigned to `self.NAME` if they are going to be used elsewhere.